src/frontend/mame/ui/miscmenu.cpp

```diff
@@ -51,6 +51,35 @@ menu_bios_selection::menu_bios_selection(mame_ui_manager &mui, render_container
 {
 }
 
+int menu_bios_selection::current_bios(device_t *device)
```

Comment:
Why is the parameter `device` a pointer rather than a reference?  The function won't work if it’s null, and it can’t be indexed like an array.

Reply:
Fixed this as well